ha/core/error.py

```diff
@@ -36,6 +36,7 @@
 HA_SYSTEM_HEALTH_FAILED         = 0x000c
 HA_CLUSTER_CLI_FAILED           = 0x000d
 HA_EVENT_ANALYZER_ERROR         = 0x000e
+HA_CLUSTER_CONFIG_ERROR         = 0x000f
```

Comment:
For test can we create separate error file  and move all error to there

`HA_CLUSTER_CONFIG_ERROR` is seems unused import

Reply:
I checked where `HA_CLUSTER_CONFIG_ERROR` is used.
It is used in exceptions `ValidationConfigError` and `ValidationStatusError` which are kind of official exceptions being returned by validator. So, i can't say that it is used only in unit tests :)